src/containers/TriggerBinding/TriggerBinding.js

```diff
@@ -0,0 +1,308 @@
+    this.props.fetchTriggerBinding({ name: triggerBindingName, namespace });
+  }
+
+  handleTabSelect = index => {
+    switch (index) {
+      case 0:
+        this.setState({
+          showYAML: false,
+          showDetails: true
+        });
+        break;
+
+      case 1:
+        this.setState({
+          showYAML: true,
+          showDetails: false
+        });
+        break;
+
+      default:
+        this.setState({
+          showYAML: false,
+          showDetails: false
+        });
+    }
+  };
+
+  render() {
+    const {
+      intl,
+      error,
+      match,
+      selectedNamespace,
+      triggerBinding
+    } = this.props;
+    const { showYAML, showDetails } = this.state;
+    const { triggerBindingName } = match.params;
+
+    if (error) {
+      return TriggerBindingContainer.notification({
+        kind: 'error',
+        intl
+      });
+    }
+    if (!triggerBinding) {
+      return TriggerBindingContainer.notification({
+        kind: 'error',
+        message: 'TriggerBinding not available',
+        intl
+      });
+    }
+
+    let labels = '';
+    const formattedLabelsToRender = [];
+    if (triggerBinding.metadata.labels) {
+      labels = JSON.stringify(triggerBinding.metadata.labels)
+        .replace('{', '')
+        .replace('}', '')
+        .replace(/['"]+/g, '')
+        .replace('$', '');
+
+      const labelsSplitOnComma = labels.split(',');
+      labelsSplitOnComma.forEach(foo => {
+        const splitOnColon = foo.split(':');
+        formattedLabelsToRender.push(`  ${splitOnColon[0]}: ${splitOnColon[1]}
+ `);
+      });
+    }
+
+    const headersForParameters = [
+      {
+        key: 'name',
+        header: intl.formatMessage({
+          id: 'dashboard.tableHeader.name',
+          defaultMessage: 'Name'
+        })
+      },
+      {
+        key: 'value',
+        header: intl.formatMessage({
+          id: 'dashboard.triggerBinding.value',
+          defaultMessage: 'Value'
+        })
+      }
+    ];
+
+    const rowsForParameters = triggerBinding.spec.params.map(
+      ({ name, value }) => ({
+        id: name,
+        name,
+        value
+      })
+    );
+
+    const emptyTextMessage = intl.formatMessage({
+      id: 'dashboard.triggerBinding.noParams',
+      defaultMessage: 'No parameters found for this TriggerBinding.'
+    });
+
+    return (
+      <div className="triggerBinding">
+        <h1>{triggerBindingName}</h1>
+        <Tabs
+          onSelectionChange={index => {
+            this.handleTabSelect(index);
+          }}
+        >
+          <Tab
+            label={intl.formatMessage({
+              id: 'dashboard.triggerBinding.details',
+              defaultMessage: 'Details'
+            })}
+          />
+          <Tab label="YAML" />
+        </Tabs>
+        {showDetails && (
+          <div className="details">
+            <p>
+              <span>
+                {intl.formatMessage({
+                  id: 'dashboard.triggerBinding.dateCreated',
```

Comment:
a lot of these strings are generic, not specific to TriggerBindings, e.g. 'Date Created', 'Details', 'Labels:'
we should reuse where possible instead of duplicating, this will keep our string count down when it comes to translation and help with consistency

Reply:
what keyword should I use instead of triggerBinding? metadata?

Reply:
for fields that are actually from the resource metadata that works 👍 

for other generic things like 'Details' maybe we go with something like 'dashboard.resource.detailsTab'?